From 430a2359473e57e5c8a27d5f363c4be34a57998e Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 29 Jun 2021 09:20:09 +0100
Subject: [PATCH 419/552] configs: Add CONFIG_MACVTAP=m (arm64 only)

From the requesting issue:

"This option is necessary for having bridge-like networking on KVM VMs
 with the host root filesystem on NFS (since regular bridge networking
 doesn't work in that case),"

See: https://github.com/raspberrypi/linux/issues/4413

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 748d8fec016c..1925a2f2ed88 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -494,6 +494,7 @@ CONFIG_DUMMY=m
 CONFIG_WIREGUARD=m
 CONFIG_IFB=m
 CONFIG_MACVLAN=m
+CONFIG_MACVTAP=m
 CONFIG_IPVLAN=m
 CONFIG_VXLAN=m
 CONFIG_NETCONSOLE=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index e86b7cb61ca5..8a8cf52258c5 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -480,6 +480,7 @@ CONFIG_DUMMY=m
 CONFIG_WIREGUARD=m
 CONFIG_IFB=m
 CONFIG_MACVLAN=m
+CONFIG_MACVTAP=m
 CONFIG_IPVLAN=m
 CONFIG_VXLAN=m
 CONFIG_NETCONSOLE=m
-- 
2.33.0

